build.gradle

```diff
@@ -6,10 +6,21 @@ buildscript {
     }
     dependencies {
         classpath 'org.shipkit:shipkit:1.0.7'
+        classpath 'com.gradle:build-scan-plugin:1.11'
         classpath 'ru.vyarus:gradle-animalsniffer-plugin:1.3.0'
     }
 }
 
+apply plugin: 'com.gradle.build-scan'
+
+buildScan {
+    if (System.getenv('CI')) {
```

Comment:
I'd say we always publish, even from local. If you are out of network, build scans degrade gracefully and scan is simply not sent. We can always turn it back to CI-only later, after we learn how it works for us.